runtime/onert/backend/cpu/KernelGenerator.cc

```diff
@@ -651,8 +651,6 @@ void KernelGenerator::visit(const ir::operation::Unpack &node)
   const auto rank = node.param().rank;
   const auto axis = ops::getAxis(rank, node.param().axis, _current_op_seq_layout);
 
```

Comment:
This condition makes error on TC.
should I put pass condition for the zero case
or is there good solution for keeping the assert condition?

Reply:
okay, I'll put pass condition for the zero case.
And keep watch for further progresses on other op's implementation.